spec/scss_converter_spec.rb

```diff
@@ -83,7 +83,7 @@ def converter(overrides = {})
     context "in safe mode" do
       let(:verter) {
         Jekyll::Converters::Scss.new(site.config.merge({
-          "sass" => {},
+          "sass" => {"load_paths" => ["bower_components/*", Dir.tmpdir]},
```

Comment:
Would you mind adding another (to-be-filtered) example here where it contains path traversal? E.g. `node_modules/*/../../../../../usr/include` or something along those lines. I believe the `sanitized_path` mapping should help with this, but better to be safe than sorry.